projects/libjpeg-turbo/Dockerfile

```diff
@@ -20,8 +20,7 @@ RUN apt-get install -y make autoconf automake libtool nasm curl
 RUN git clone --depth 1 https://github.com/libjpeg-turbo/libjpeg-turbo
 
 RUN mkdir afl-testcases
-RUN curl -o afl-testcases/afl_testcases.tgz http://lcamtuf.coredump.cx/afl/demo/afl_testcases.tgz
-RUN cd afl-testcases/ && tar -xf afl_testcases.tgz
+RUN cd afl-testcases/ && curl http://lcamtuf.coredump.cx/afl/demo/afl_testcases.tgz | tar -xz
```

Comment:
To save on image size consider combining the next RUN too. This way this would be only one layer and temp files won't be stored:

```
RUN mkdir xxx && \
        cd && \
        ... && \
       rm tempfiles
```